tests/csapi/power_levels_test.go

```diff
@@ -40,3 +43,83 @@ func TestDemotingUsersViaUsersDefault(t *testing.T) {
 		},
 	})
 }
+
+func TestPowerLevels(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintAlice)
+	defer deployment.Destroy(t)
+
+	alice := deployment.Client(t, "hs1", "@alice:hs1")
+
+	roomID := alice.CreateRoom(t, map[string]interface{}{})
+
+	// sytest: GET /rooms/:room_id/state/m.room.power_levels can fetch levels
+	t.Run("GET /rooms/:room_id/state/m.room.power_levels can fetch levels", func(t *testing.T) {
```

Comment:
This test appears to be missing `assert_json_keys( $body, qw( ban kick redact state_default events_default users_default ));` and a few other checks
https://github.com/matrix-org/sytest/blob/fd4f4132913e9575471af515f67c633d2e3123d3/tests/10apidoc/36room-levels.pl#L17-L34